### PR TITLE
Added package logs link and changed chaind card color

### PR DESCRIPTION
**Pull request by pablomendezroyo**
Changes of chain card when error
- Add package logs path as Link
- CHange from danger to warning

### CODE REVIEW

File: packages/admin-ui/src/pages/dashboard/components/ChainCard.tsx [hunk 1]
```diff
@@ -37,13 +39,16 @@ function ChainCard(chain: ChainData) {
...
-        <ProgressBar now={100} variant="danger" />
+        <ProgressBar now={100} variant="warning" />
       ) : (
         <ProgressBar now={100} variant="success" />
       )}
 
       <div className="message">
         <RenderMarkdown source={message} noMargin />
+        {error ? (
+          <a href={path.join(apiUrl, dnpName, "logs")}>More info</a>
```
Comment (dapplion) [inline packages/admin-ui/src/pages/dashboard/components/ChainCard.tsx:50]: You must never use <a> links in a React site. You have to use programatic navigation with `react-router` and relative links

**Review by dapplion — CHANGES_REQUESTED** [review]

File: packages/admin-ui/src/pages/dashboard/components/ChainCard.tsx (hunk 1 above)
Comment (pablomendezroyo) [reply packages/admin-ui/src/pages/dashboard/components/ChainCard.tsx]: done with last commit

File: packages/admin-ui/src/pages/dashboard/components/ChainCard.tsx [hunk 2]
```diff
@@ -46,9 +45,7 @@ function ChainCard(chain: ChainData) {
 
       <div className="message">
         <RenderMarkdown source={message} noMargin />
-        {error ? (
-          <a href={path.join(apiUrl, dnpName, "logs")}>More info</a>
-        ) : null}
+        {error ? <Link to={`/packages/${dnpName}/logs`}>More info</Link> : null}
```
Comment (dapplion) [inline packages/admin-ui/src/pages/dashboard/components/ChainCard.tsx:48]: Check other parts of the UI how to import the `/packages` path from pages/packages/data. Also use urlJoin

Comment (pablomendezroyo) [reply packages/admin-ui/src/pages/dashboard/components/ChainCard.tsx]: done in last commit

**Review by dapplion — APPROVED** [review]

**Pull request merged**